recipes/cpp_project_framework/all/conanfile.py

```diff
@@ -0,0 +1,99 @@
+from conan import ConanFile
+from conans import CMake, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+
+required_conan_version = ">=1.33.0"
+
+
+class CppProjectFrameworkConan(ConanFile):
+    name = "cpp_project_framework"
+    license = "GNU Affero General Public License v3.0"
```

Comment:
We like to use SPDX identifiers https://github.com/conan-io/conan-center-index/blob/master/docs/reviewing.md#license-attribute

If i remember correctly... 

```suggestion
    license = "AGPL-3.0"
```